flink-table/flink-sql-client/src/test/resources/sql/table.q

```diff
@@ -79,7 +79,10 @@ show tables;
 
 # test SHOW CREATE TABLE
 show create table orders;
-CREATE TABLE `default_catalog`.`default_database`.`orders` (
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Is this by design? Personally, I felt the column length be better calculated according to the content length. Besides, the right border of the table is missing?
![image](https://user-images.githubusercontent.com/55568005/213472289-dcaccb1f-9410-494f-b77d-dc033f3eae80.png)

I thought it should be the following format that aligned with what we have now.

```text
Flink SQL> show create table foo;
+-----------------------------------------------------------+
|                                                    result |
+-----------------------------------------------------------+
| CREATE TABLE `default_catalog`.`default_database`.`foo` ( |
|  `f0` INT,                                                |
|  `f1` VARCHAR(2147483647)                                 |
| ) WITH (                                                  |
|  'connector' = 'datagen'                                  |
| )                                                         |
+-----------------------------------------------------------+  
1 row in set
```

Reply:
![image](https://user-images.githubusercontent.com/55568005/213477033-23014f61-46e8-44da-903b-232f7cdf66f4.png)